tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1606,28 +1606,35 @@ class OpConverterTest : public ::testing::Test {
   }
 
   // Helper method to run both validation and conversion, and check the output
-  // shape.
-  void RunValidationAndConversion(const NodeDef& node_def, const Status& status,
-                                  const char* output_name,
-                                  const std::vector<int>& exp_out_dims) {
+  // shape. TODO(tfeher have variant that can check multiple outputs
```

Comment:
Let's fix the format for TODO and the description:
TODO(tfeher): Add a variant to check multiple outputs.

Reply:
Actually this TODO is out of date, the modified code already checks multiple outputs. I have removed it.